app/models/user.rb

```diff
@@ -7,5 +7,5 @@ class User < ApplicationRecord
   # TODO: amir - Validate email format.
   validates :email, presence: true, uniqueness: { case_sensitive: false, scope: :provider }
   validates :provider, presence: true
-  validates :password_confirmation, presence: true
+  validates :password_confirmation, presence: true, on: :create
```

Comment:
Could you explain a bit why this needed to be added?

Reply:
Will we be having a separate page to change the password like with GLv2?

Reply:
You got reason,